### PR TITLE
Rename dca to tarc

### PR DESCRIPTION
**Pull request by rawOrlando**
This changes dca.tufts.edu tarc.tufts.edu to
     But does not touch dca.tufts.edu/aas

And changes "Digital Collections and Archives" and slight variations on that to "Tufts Archival Research Center"

### CODE REVIEW

File: config/predicate_mappings.yml [hunk 1]
```diff
@@ -46,7 +46,7 @@
     :has_model: hasModel
   http://www.openarchives.org/OAI/2.0/: 
     :oai_item_id: itemID
-  http://dca.lib.tufts.edu/ontology/rcr#:
+  http://tarc.lib.tufts.edu/ontology/rcr#:
```
[the remark this answers is not in the document]
Comment (rawOrlando) [reply config/predicate_mappings.yml:49]: no

File: public/staff.html [hunk 2]
```diff
@@ -74,15 +74,15 @@
                     </td></tr>
                     <tr>
                     <td><br/><br/>
-                        <a href="http://dca.tufts.edu/general/staff.html"><b>Tufts University <br/>
-                            Digital Collections and Archives (DCA)</b></a>
+                        <a href="http://tarc.tufts.edu/general/staff.html"><b>Tufts University <br/>
```
Comment (rawOrlando) [inline public/staff.html:77]: This link 404s what should I replace it with?

Comment (mkorcy) [reply public/staff.html:77]: https://tarc.tufts.edu/about/staff

**Review by mkorcy — APPROVED** [review]
👍

**Pull request merged**